crates/nu-command/src/formats/from/ini.rs

```diff
@@ -58,23 +59,50 @@ pub fn from_ini_string_to_value(
     span: Span,
     val_span: Span,
 ) -> Result<Value, ShellError> {
-    let v: Result<IndexMap<String, IndexMap<String, String>>, serde_ini::de::Error> =
-        serde_ini::from_str(&s);
-    match v {
-        Ok(index_map) => {
-            let (cols, vals) = index_map
-                .into_iter()
-                .fold((vec![], vec![]), |mut acc, (k, v)| {
-                    let (cols, vals) = v.into_iter().fold((vec![], vec![]), |mut acc, (k, v)| {
-                        acc.0.push(k);
-                        acc.1.push(Value::String { val: v, span });
-                        acc
+    let ini: Result<RustIni::Ini, RustIni::ParseError> = RustIni::Ini::load_from_str(&s);
```

Comment:
Hi, I don't think we need `extern crate ini as RustIni` line.

Then we can just rename the `ini` variable to `ini_config`

```rust
let ini_config: Result<ini::Ini, ini::ParseError> = ini::load_from_str(&s);
```

Reply:
Hi @WindSoilder, 
Thank you for the review, updated PR with suggested changes.